images/ingress-nginx-controller/configs/latest.apko.yaml

```diff
@@ -0,0 +1,196 @@
+contents:
+  packages:
+    - curl
+    - curl-dev
+    - bash
+    - apk-tools
+    - busybox
+    - git
+    - build-base
+    - ca-certificates-bundle
+    - ca-certificates
+    - openssl
+    - openssl-dev
+    - ingress-nginx-controller
+    - dumb-init
+    - geoip-dev
+    - pcre
+    - zlib
+    - libxml2
+    - tzdata
+    - opentracing-cpp
+    - msgpack
+    - mimalloc=1.7.6-r2
+    - datadog-cpp
+    - yaml-cpp
+    - yaml-cpp-dev
+    - zipkin-cpp
+    - modsecurity
+    - lua-resty-balancer
+    - lua-cjson
+    - luajit
+    - luajit-dev
+    - lua-resty-cache
+    - lua-resty-cookie
+    - lua-resty-dns
+    - lua-resty-core
+    - lua-resty-global-throttle
+    - lua-resty-ipmatcher
+    - lua-resty-memcached
+    - lua-resty-http
+    - lua-resty-lock
+    - lua-resty-redis
+    - lua-resty-string
+    - lua-resty-upload
```

Comment:
maybe these packages can be dropped once [this](https://github.com/wolfi-dev/os/pull/3784/files#diff-433c67391bb6d11409699d4634e68ebd1d199daf95eaf43b0fc745f3a6014927R11-R47) merges as there are runtime deps defined on the main package?